cmd/next.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+var (
+	errNotSupportedPart = errors.New("part string is not a supported next version increment")
+)
+
+var nextCmd = &cobra.Command{
+	Use:   "next part",
+	Short: "Next increment version based on semantic versioning",
```

Comment:
Just want to clarify next does not bump it only outputs the next value.

```suggestion
	Short: "Next echos the next version based on semantic versioning",
```